packages/turbo-workspaces/src/install.ts

```diff
@@ -68,6 +68,18 @@ export const PACKAGE_MANAGERS: Record<
       semver: ">=2",
     },
   ],
+  bun: [
+    {
+      name: "bun1",
```

Comment:
I'm comfortable with this being just `bun` since it doesn't end up exposed to users or constrain us from renaming to `bun1` later.